dev-libs/libkrunfw/libkrunfw-4.3.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A dynamic library bundling the guest payload consumed by libkrun"
+HOMEPAGE="https://github.com/containers/libkrunfw"
+
+KERNEL_VERSION=linux-6.6.44
+
+SRC_URI="
+	https://cdn.kernel.org/pub/linux/kernel/v6.x/${KERNEL_VERSION}.tar.xz
+	https://github.com/containers/libkrunfw/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+"
+
+LICENSE="GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm64"
+BDEPEND="
+	app-alternatives/cpio
+	dev-lang/perl
+	app-alternatives/bc
+	dev-build/make
+	sys-devel/bison
+	sys-devel/flex
+	>=sys-libs/ncurses-5.2
+	virtual/libelf
+	virtual/pkgconfig
+	dev-python/pyelftools
```

Comment:
I assume it uses a tool that pyelftools installs and doesn't import a Python library? (If so, it would need python-any-r1).